renderdoc/driver/vulkan/vk_info.cpp

```diff
@@ -1037,6 +1037,17 @@ void VulkanCreationInfo::RenderPass::Init(VulkanResourceManager *resourceMan,
     if(separateStencil)
       dst.stencilLayout = separateStencil->stencilLayout;
 
+    // VK_KHR_depth_stencil_resolve
+    const VkSubpassDescriptionDepthStencilResolveKHR *depthstencilResolve =
+        (const VkSubpassDescriptionDepthStencilResolveKHR *)FindNextStruct(
```

Comment:
Generally I prefer using the promoted structs/types in all cases - here you're using the promoted core `VK_STRUCTURE_TYPE_SUBPASS_DESCRIPTION_DEPTH_STENCIL_RESOLVE` which is good, but the struct should be `VkSubpassDescriptionDepthStencilResolve` to match. Leaving the comment to reference the extension is still fine though, since 'version 1.2' is less specific and it's easy to go from extension -> core version.

Reply:
Got it, should be fixed now. Thanks for reviewing!